stopwords.txt

```diff
@@ -0,0 +1,121 @@
+a
```

Comment:
Out of curiosity, where did these come from? Just trial and error?

Reply:
I got the stopwords from CSC 202. I also added some more "stopwords" since they were problematic.